File: Programmers/pro_77484_로또의 최고 순위와 최저 순위.py
def solution(lottos, win_nums):
    answer = []
    zeroNum=0
    correctNum=0
    for i in lottos:
        if(i==0):
            zeroNum+=1
        else:
            if i in win_nums:
                correctNum+=1
    answer.append(min(6,7-(correctNum+zeroNum)))
    answer.append(min(6,7-correctNum))
    return answer

File: Programmers/pro_77484_로또의최고순위와최저순위.js
function solution(lottos, win_nums) {
    const rank = [6, 6, 5, 4, 3, 2, 1]

    let zeroCnt = lottos.filter((v) => !v).length
    let minCnt = lottos.filter((v) => win_nums.includes(v)).length

    let maxCnt = minCnt + zeroCnt

    return [rank[maxCnt], rank[zeroCnt]];
}